services/common/src/components/forms/RenderFileUpload.spec.tsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import { render } from "enzyme";
+import RenderFileUpload from "./RenderFileUpload";
+import { Provider } from "react-redux";
+import { configureStore } from "@reduxjs/toolkit";
+import { sharedReducer } from "@mds/common/redux/reducers/rootReducerShared";
```

Comment:
I would suggest using the redux wrapper instead so that if we ever update the reducer setup or change the tests to use an initial state, that we won't have to redo all the imports. Which I guess is only on core/MS right now.